### ISSUE TITLE
Heartbeat code complains and kills the program on perfectly good connections

### ISSUE THREAD

**Issue by kuba-felendzer** [zach-pike/nodeMessenger#1]
The new heartbeat code, which is meant to check if the server is dead or not, kills the process after a couple seconds unless the user spams text. I've resolved this issue by literally commenting out the part of the code that kills the process, and the messaging works just fine.

**Comment by zach-pike** [comment]
Is the server you are trying to connect to running?

**Comment by kuba-felendzer** [comment]
Yes, the server and chat functionality is working fine. I commented out the line that kills the process, and the chat worked fine. I had at least 3 people, one hosting the server, chatting through the thing at once after we removed that line.
